tests/phpunit/tests/tm/test-wpml-pb-handle-custom-fields.php

```diff
@@ -210,4 +215,47 @@ public function it_does_not_copy_custom_fields_when_original_custom_field_does_n
 
 		$subject->copy_custom_fields( $new_post_id, $original_post_id );
 	}
+
+	/**
+	 * @test
+	 */
+	public function it_copies_fields() {
+		$original = 1;
+		$new = 2;
+		$field = 'field_name';
+		$data = 'some data';
+
+		\WP_Mock::userFunction( 'get_post_meta',
+			[
+				'args' => [ $original, $field, true ],
+				'return' => $data
+			]
+		);
+
+		\WP_Mock::userFunction( 'update_post_meta',
+			[
+				'times' => 1,
+				'args' => [ $new, $field, $data ],
+			]
+		);
+
+		$slash_json_mock = FunctionMocker::replace( 'WPML_PB_Handle_Custom_Fields::slash_json', $data );
+
+		WPML_PB_Handle_Custom_Fields::copy_field( $new, $original, $field );
+
+		$slash_json_mock->wasCalledWithOnce( [ $data ] );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_slashes_json() {
+		$not_json = 'not json';
+		$this->assertEquals( $not_json, WPML_PB_Handle_Custom_Fields::slash_json( $not_json ) );
+
+		$json = [ 'data' => 'something' ];
+		$json_string = json_encode( $json );
+
+		$this->assertEquals( addslashes( $json_string ), WPML_PB_Handle_Custom_Fields::slash_json( $json_string ) );
+	}
```

Comment:
@BruceOnTheGo  These are two separate tests. 

The test name says that it "will slashes json", which I translate into "it will slash a JSON object", but the first test does quite the opposite.

I would split it into `it_slash_a_json_object` and `it_does_NOT_slash_a_string`?

Reply:
Done